back-office/shop-parameters/order-settings/order-setting-statuses-page.md

```diff
@@ -61,30 +180,150 @@ Note that it is not possible to delete default order statuses such as Payment ac
+
+1) Checkbox: The checkbox isn't displayed for the default order return status.
+
+There are 3 columns in the grid:
+
+1) Id
+
+It's possible to sort by id asc or desc and search by id.
+
+2) Name 
+
+It's possible to sort by name asc or desc and search by name.
+
+The order return status names are displayed with their color in background.
+
+3) Actions
+
+There are two actions:
+
+- Edit, the main one:
+
+After clicking on it, the order return status is opened in the same tab.
+This action can also be performed by clicking on the line.
+
+- Delete, the second one:
+
+This option isn't displayed for the default order return statuses and the order return statuses added by modules.
+
+After clicking on it, a modal is opened asking to confirm or cancel the action: "Are you sure you want to delete the selected item(s)?"
+
+The action can be canceled by clicking on the cross or on the cancel button.
+
+After clicking on "Delete" button, the message is deleted and a successful message is displayed "Successful deletion."
+
+#### Search
+
+Below, the same behavior as others listing:
+
+By default, the search button is disabled. It becomes enabled when something is filled in a search field.
+
+After clicking on "Search", the search is applied and the order statuses matching with the search are displayed
+
+If no results are found, then a message "No records found" is displayed.
+
+After clicking on "Search", this button is disabled and a "Reset" button is displayed.
+
+After clicking on "Reset", the search is removed and all order statuses are displayed.
+
+#### Pagination
+
+The pagination of the list starts when there are more than 20 order return statuses.
+
+The merchant can select the number of order return statuses displayed through a drop-down: 20, 50, 300, 1000
+
+When there are more results than the number selected, the merchant can navigate through pages.
+
+If a sort and/or a search have been done, results should be kept when moving on to the next page.
+
+#### Default order return statuses
+
+By default, PrestaShop has the following order return statuses: 
+- id 1:	Waiting for confirmation	
+- id 2:	Waiting for package	
+- id 3:	Package received	
+- id 4:	Return denied	
+- id 5:	Return completed	
 
 ![Return statuse list](/img/list%20return%20statuses.png)
 
 **The first return status generated** in the return workflow is **“Waiting for confirmation”** when the customer creates a return demand in the Front Office. 
 
-**Adding or Editing a return status** allows the user to add or edit **the status name** input text on any back-office language and **the label color** with a color picker or directly into the input text with the hexadecimal code. 
+### Add / edit order return status
+
+**Adding or Editing an order return status** allows the user to add or edit **the order return status name** input text on any back-office language and **the label color** with a color picker or directly into the input text with the hexadecimal code. 
```

Comment:
What should happen when the color field is left blank ? (maybe it's out of scope for this PR)

Reply:
I added the default value of the field: #ffffff
There is an issue when the color is not defined: https://github.com/PrestaShop/PrestaShop/issues/21284
I linked the issue, the behavior in this case is still be defined